helm-chart/renku/values.yaml

```diff
@@ -126,6 +136,26 @@ global:
       tag: "0.0.2"
     customCAs: []
       # - secret:
+  
+  ## Values for the RabbitMQ setup.
+  rabbitmq:
+    ## Whether to install RabbitMQ as part of the Renku helm chart.
+    ## If set to false, then an external RabbitMQ instance needs to be available
+    ## and the external section below should be filled out.
+    enabled: true
+    ## The external section is relevant only if an external RabbitMQ will be used
+    ## with Renku. The RabbitMQ management plugin needs to be enabled so that the
+    ## proper vhost, users and permissions are setup.
+    external:
+      managementUrl:
```

Comment:
Is there a reason this is `managementUrl`, and not just `URL` or `address`?

Reply:
we should probably also allow the port to be specified for external rabbitmq instances

Reply:
okay, so this isn't actually the address used for interacting with the message service?

Reply:
Not really - the messaging service is exposed on a particular port which speaks AMQP and the management interface is a http interface for looking at system state and I think it can be used for some control operations.

Reply:
Ah okay. I misunderstood this then.